data.csv

```diff
@@ -1,7 +1,3 @@
-﻿"Data Source","World Development Indicators",
-
-"Last Updated Date","2018-10-18",
-

```

Comment:
How might you approach parsing the CSV in future without altering the original dataset?